content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -22,61 +23,62 @@ This document describes the current state of `VolumeSnapshots` in Kubernetes. Fa
 
 #### Dynamic
-When none of the static `VolumeSnapshotContents` the administrator created matches a user's `VolumeSnapshot`,
+When none of the static `VolumeSnapshotContent` resources the administrator created matches a user's `VolumeSnapshot`,
 the cluster may try to dynamically provision a volume snapshot specially for the `VolumeSnapshot` object.
-This provisioning is based on `VolumeSnapshotClasses`: the `VolumeSnapshot` must request a
+This provisioning is based on `VolumeSnapshotClass` resources: the `VolumeSnapshot` must request a
 [volume snapshot class](/docs/concepts/storage/volume-snapshot-classes/) and
 the administrator must have created and configured that class in order for dynamic
 provisioning to occur.
 
 ### Binding
 
-A user creates, or has already created in the case of dynamic provisioning, a `VolumeSnapshot` with a specific amount of storage requested and with certain access modes. A control loop watches for new VolumeSnapshots, finds a matching VolumeSnapshotContent (if possible), and binds them together. If a VolumeSnapshotContent was dynamically provisioned for a new VolumeSnapshot, the loop will always bind that VolumeSnapshotContent to the VolumeSnapshot. Once bound, `VolumeSnapshot` binds are exclusive, regardless of how they were bound. A VolumeSnapshot to VolumeSnapshotContent binding is a one-to-one mapping.
+A user creates, or has already created in the case of dynamic provisioning, a `VolumeSnapshot` with a specific set of parameters. A control loop watches for new `VolumeSnapshot` requests, finds a matching `VolumeSnapshotContent` (if possible), and binds them together. If a `VolumeSnapshotContent` was dynamically provisioned for the new `VolumeSnapshot`, the loop will always bind that `VolumeSnapshotContent` to the `VolumeSnapshot`. Once bound, `VolumeSnapshot` binds are exclusive, regardless of how they were bound. A `VolumeSnapshot` to `VolumeSnapshotContent` binding is a one-to-one mapping.
 
-VolumeSnapshots will remain unbound indefinitely if a matching VolumeSnapshotContent does not exist. VolumeSnapshots will be bound as matching VolumeSnapshotContents become available.
+*Volume snapshots* will remain unbound indefinitely if a matching `VolumeSnapshotContent` does not exist. *Volume snapshots* will be bound as matching `VolumeSnapshotContent` resources become available.
 
 ### Persistent Volume Claim in Use Protection
 
-The purpose of the Persistent Volume Claim Object in Use Protection feature is to ensure that in-use PVC API objects are not removed from the system (as this may result in data loss).
+The purpose of the *Persistent Volume Claim Object in Use* Protection feature is to ensure that in-use PVC API objects are not removed from the system (as this may result in data loss).
```

Comment:
Should we clarify in this paragraph that snapshots is extending that feature? It's a little confusing because the in-tree feature uses the same name. Or should we even call it a different feature? Is it using the same finalizer or do we add another finalizer for snapshots?

Reply:
It is a different finalizer with a different key added on PVC.  In the code, this is also called SnapshotSourceFinalizer.  We can call this "Snapshot Source in Use Protection" to differentiate?

const PVCFinalizer = "snapshot.storage.kubernetes.io/pvc-protection"

Reply:
How about "Protecting PVCs during Snapshotting"?